AppInspector.CLI/CLICmdOptions.cs

```diff
@@ -2,46 +2,42 @@
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
 using CommandLine;
-using NLog;
+using Microsoft.ApplicationInspector.Commands;
 
-namespace Microsoft.ApplicationInspector.Commands
+namespace Microsoft.ApplicationInspector.CLI
 {
-    public class AllCommandOptions
+    /// <summary>
+    /// CLI command option classes add output arguments to common properties for each command verb
+    /// </summary>
+    /// 
+    public class CLICommandOptions : CommandOptions
     {
-        public Logger Log { get; set; }
+        [Option('o', "output-file-path", Required = false, HelpText = "Output file path")]
+        public string OutputFilePath { get; set; }
+        [Option('f', "output-file-format", Required = false, HelpText = "Output format [json|text]", Default = "text")]
+        public string OutputFileFormat { get; set; }
 
-        [Option('l', "log-file-path", Required = false, HelpText = "Log file path")]
-        public string LogFilePath { get; set; }
-        [Option('v', "log-file-level", Required = false, HelpText = "Log file level [Debug|Info|Warn|Error|Trace|Fatal|Off]", Default = "Error")]
-        public string LogFileLevel { get; set; }
+        public CLICommandOptions()
+        {
+            OutputFileFormat = "text";
```

Comment:
I would recommend built in default declarations functionality in the option definition or even `{ get; set; } = value` the property declaration over hardcoded values in the empty `.ctor`

Reply:
sweet trick will get on it